openupgrade_scripts/scripts/base_automation/14.0.1.0/post-migration.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+
+def fill_base_automation_ir_model_fields_rel(env):
+    openupgrade.logged_query(
+        env.cr,
+        sql.SQL(
+            """
+        WITH q1 AS (
+            SELECT ba.id, unnest(
+                string_to_array(ba.{old_field}, ',')
+            ) AS field, ias.model_id
+            FROM base_automation ba
+            JOIN ir_act_server ias ON ba.action_server_id = ias.id
+            WHERE ba.on_change_fields is not null
+        )
+        INSERT INTO base_automation_onchange_fields_rel
+            (base_automation_id, ir_model_fields_id)
+        SELECT q1.id, imf.id
+        FROM ir_model_fields AS imf
+        JOIN q1 ON imf.name = q1.field
+        WHERE imf.model_id = q1.model_id
+        """
+        ).format(
+            old_field=sql.Identifier(openupgrade.get_legacy_name("on_change_fields")),
+        ),
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_moved_fields(
+        env.cr,
+        "base.automation.lead.test",
+        [
+            "___last_update",
+            "_order",
+            "active",
+            "date_action_last",
+            "deadline",
+            "display_name",
+            "employee",
+            "is_assigned_to_admin",
+            "line_ids",
+            "name",
+            "partner_id",
+            "priority",
+            "state",
+            "user_id",
+        ],
+        "base_automation",
+        "test_base_automation",
+    )
+    openupgrade.update_module_moved_fields(
+        "base.automation.line.test",
```

Comment:
```suggestion
        env.cr,
        "base.automation.line.test",
```

Reply:
There's no need anymore to use `update_module_moved_fields` since v13, so you can remove this and please do a PR saying so on the docstring of the method.